packages/vscode/package.json

```diff
@@ -1,19 +1,46 @@
 {
-  "name": "@zowe/secrets-for-kubernetes-for-zowe-explorer",
+  "name": "zowe-secrets-for-kubernetes",
   "version": "1.0.0",
-  "scripts": {
-    "build": "tsc --pretty",
-    "test": "echo \"VS Code package: No test specified\" && exit 0",
-    "test:system": "echo \"VS Code package: No system test specified\" && exit 0",
-    "test:unit": "echo \"VS Code package: No unit test specified\" && exit 0"
-  },
-  "main": "lib/index.js",
+  "repository": "https://github.com/zowe/zowe-cli-secrets-for-kubernetes",
+  "main": "dist/extension.js",
+  "typings": "dist/extension.d.ts",
   "keywords": [
     "credential manager",
     "secure store",
     "password"
   ],
+  "publisher": "Zowe",
   "author": "IBM",
   "license": "EPL-2.0",
-  "dependencies": {}
+  "engines": {
+    "vscode": "^1.76.0"
+  },
+  "scripts": {
+    "build": "tsup",
+    "dev": "npm run build -- --watch",
+    "test": "echo \"VS Code package: No test specified\" && exit 0",
+    "test:system": "echo \"VS Code package: No system test specified\" && exit 0",
+    "test:unit": "echo \"VS Code package: No unit test specified\" && exit 0",
+    "vscode:prepublish": "npm run clean && tsup",
+    "clean": "rimraf dist",
+    "package": "vsce package --allow-star-activation --no-yarn --out ../../dist/Zowe.secrets-for-kubernetes.vsix"
+  },
+  "activationEvents": [
+    "*"
```

Comment:
I know the VS Code extension is just a sample for now, but something to consider when we build out the full extension - can we remove the `*` activation event?

Microsoft suggests to avoid using it if possible in [their docs](https://code.visualstudio.com/api/references/activation-events#Start-up):
> **Note:** To ensure a great user experience, please use this activation event in your extension only when no other activation events combination works in your use-case.

Perhaps we could remove this activation event, and instead make Zowe Explorer trigger activation of a credential manager extension when it finds one installed?